the-collab-lab/tcl-20-whats-near-me
# Ss dt api setup

## Description

-Made call to Wikipeadia api through provided proxy endpoint
-Rendered location pins on map based on response from api call
-Added inconify package to project for easy acess icon components 


## Related Issue



## Acceptance Criteria


- [x] Pins should be 24px x 24px in size

- [x] Pins should have a little down caret on the bottom edge of the pin that points to the location of the place

- [x] Pins should show a scaled-down (object-fit: cover style) thumbnail image in cases when a thumbnail image is present

- [x] Pins should render a default pin image when no thumbnail is returned

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|  ✓   | :link: Update dependencies |
|     | :scroll: Docs              |

### Before
![image](https://user-images.githubusercontent.com/17222004/104822567-63623180-5811-11eb-8f48-9a368b9c188a.png)




### After
![image](https://user-images.githubusercontent.com/17222004/104822609-db305c00-5811-11eb-80fc-48283816d8a6.png)







## Testing Steps / QA Criteria


Updating the location coordinates in the api call and adjusting the centering parameters in the GoogleMap React component should render pins for neraby locations returned from the api. Check the components section in dev tools to see that the api response has been saved to state.
